doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -588,16 +335,12 @@ Trusted Firmware-M
   .. note::
      This can be a breaking change if the UART instance used by TF-M is already in use, for example by modem trace with a UART backend.
 
-cJSON
-=====
-
-|no_changes_yet_note|
-
 Documentation
 =============
 
 * Updated:
 
+  * Replaced the Welcome to the |NCS| page with the existing :ref:`ncs_introduction` page.
```

Comment:
May be:
```suggestion
  * Removed the Welcome to the |NCS| page.
    This page is replaced with existing :ref:`ncs_introduction` page.
```
As a new bullet point. This is to match with release notes structure we have.